### ISSUE TITLE
0.26 transcoding permanently queued

### ISSUE THREAD

**Issue by RXM307** [roadrunner6/mythcut#5]
Jobs get queued permanently, appears to be date related most likely due to the utc time changes implemented in this new version ill poke around and see if I can find a fix

**Comment by RXM307** [comment]
Yep as I thought it is queuing jobs in the past, the quick and dirty hack is to replace the last CURRENT_TIMESTAMP in lib/Movie.php with UTC_TIMESTAMP however maybe a more eligant solution is to detect if mythtv is version 0.26 or above and set this automatically?

**Comment by roadrunner6** [comment]
I'm still stuck with 0.25. My schema version is 1299, and i guess it's trivial to check the schema version there. What schema version do you have in 0.26?

**Comment by RXM307** [comment]
According to http://www.mythtv.org/wiki/Category:DB_Table 1307

I also noticed the times are also incorrect in the Movie select etc, which files would I need to change to fix these also?

**Comment by RXM307** [comment]
My Fork has the UTC_TIMESTAMP change above https://github.com/RXM307/mythcut
